content/guides/getting-started.md

```diff
@@ -246,7 +246,7 @@ Version: webpack 2.2.0
 Time: 390ms
     Asset    Size  Chunks                    Chunk Names
 bundle.js  544 kB       0  [emitted]  [big]  main
-   [0] ./~/lodash/lodash.js 540 kB {0} [built]
+   [0] ./src/index.js 219 bytes {0} [built]
    [1] (webpack)/buildin/global.js 509 bytes {0} [built]
    [2] (webpack)/buildin/module.js 517 bytes {0} [built]
    [3] ./src/index.js 278 bytes {0} [built]
```

Comment:
`./src/index.js` is already mentioned at the bottom of these three examples so either that should be replaced with the `+ 1 hidden module` or we should re-evaluate this change. What version of webpack are you on? I think maybe we should just add a warning at the top of this guide or in `/guides/index.md` to indicate that output might vary between versions of webpack -- trying to always keep these up to date will not be easy.

cc @TheDutchCoder